test/qpu_interface.jl

```diff
@@ -511,6 +511,72 @@ end
 
 end
 
+@testset "AbstractConnectivity: isequal" begin
+
+    @test isequal(LineConnectivity(4), LineConnectivity(4))
+    @test !isequal(LineConnectivity(4), LineConnectivity(5))
+
+    @test !isequal(
+        LineConnectivity(4),
+        Snowflurry.with_excluded_positions(LineConnectivity(4), [1, 2, 3]),
+    )
+    @test !isequal(
+        Snowflurry.with_excluded_positions(LineConnectivity(4), [1, 2, 3]),
+        Snowflurry.with_excluded_positions(LineConnectivity(4), [4]),
+    )
+    @test isequal(
+        Snowflurry.with_excluded_positions(LineConnectivity(4), [1, 2, 3]),
+        Snowflurry.with_excluded_positions(LineConnectivity(4), [1, 2, 3]),
+    )
+
+    @test !isequal(
+        LineConnectivity(4),
+        Snowflurry.with_excluded_connections(LineConnectivity(4), [(1, 2)]),
+    )
+    @test !isequal(
+        Snowflurry.with_excluded_connections(LineConnectivity(4), [(1, 2), (2, 3)]),
+        Snowflurry.with_excluded_connections(LineConnectivity(4), [(1, 2)]),
+    )
+    @test isequal(
+        Snowflurry.with_excluded_connections(LineConnectivity(4), [(1, 2), (2, 3)]),
+        Snowflurry.with_excluded_connections(LineConnectivity(4), [(1, 2), (2, 3)]),
+    )
+
+    @test !isequal(LatticeConnectivity(4, 3), LineConnectivity(4))
+
+    @test isequal(LatticeConnectivity(4, 3), LatticeConnectivity(4, 3))
+    @test !isequal(LatticeConnectivity(4, 3), LatticeConnectivity(4, 4))
+    @test !isequal(LatticeConnectivity(4, 3), LatticeConnectivity(3, 3))
+
+    @test !isequal(
+        LatticeConnectivity(4, 3),
+        Snowflurry.with_excluded_positions(LatticeConnectivity(4, 3), [1, 2, 3]),
+    )
+    @test !isequal(
+        Snowflurry.with_excluded_positions(LatticeConnectivity(4, 3), [1, 2, 3]),
+        Snowflurry.with_excluded_positions(LatticeConnectivity(4, 3), [4]),
+    )
+    @test isequal(
+        Snowflurry.with_excluded_positions(LatticeConnectivity(4, 3), [1, 2, 3]),
+        Snowflurry.with_excluded_positions(LatticeConnectivity(4, 3), [1, 2, 3]),
+    )
+
+    @test !isequal(
+        LatticeConnectivity(4, 3),
+        Snowflurry.with_excluded_connections(LatticeConnectivity(4, 3), [(1, 4)]),
+    )
+    @test !isequal(
+        Snowflurry.with_excluded_connections(LatticeConnectivity(4, 3), [(1, 4), (2, 4)]),
+        Snowflurry.with_excluded_connections(LatticeConnectivity(4, 3), [(1, 4)]),
+    )
+    @test isequal(
+        Snowflurry.with_excluded_connections(LatticeConnectivity(4, 3), [(1, 4), (2, 4)]),
+        Snowflurry.with_excluded_connections(LatticeConnectivity(4, 3), [(1, 4), (2, 4)]),
+    )
```

Comment:
Could add a test with unsorted excluded connections.

Reply:
added

Reply:
You actually forgot to add the test.

Reply:
haha yes, ok it's added now